source/includes/substitute-placeholders-azure.rst

```diff
@@ -0,0 +1,26 @@
+.. note:: Placeholder Text
+
+    You must substitute all text in quotes and angle brackets with
+    your KMS configuration values.
+
+    For example, the Node.js code prompts you to include a client id value as
+    follows:
+
+    .. code-block:: javascript
+       :copyable: false
+
+       azure: {
+         clientId: "<Azure client ID>"
+         ...
+       }
+
+    If your Azure client ID is "12345678", you should substitute the text as
```

Comment:
[suggestion] ID is capitalized here but lowercase on line 6 and in the AWS note - suggest lowercasing this one (or capitalizing the other two) for consistency